service/src/test/java/com/sedliarov/learningtable/utils/GroupFixture.java

```diff
@@ -47,49 +45,49 @@ public Group createEntityWithName(String name) {
   /**
    * Method for create Group entity and set teacher.
    */
-  public Group createEntityWithTeacher(Teacher teacher) {
+  public Group createEntityWithTeacher(UUID teacher) {
```

Comment:
And the same for other
```suggestion
  public Group createEntityWithTeacher(UUID teacherId) {
```

Reply:
Done